src/Nethermind/Nethermind.Specs.Test/ChainSpecStyle/ChainSpecBasedSpecProviderTests.cs

```diff
@@ -821,6 +822,80 @@ public static IEnumerable BlockNumbersAndTimestampsNearForkActivations
         }
     }
 
+    [TestCaseSource(nameof(BlobScheduleActivationsTestCaseSource))]
+    public void Test_BlobSchedule_IsApplied_AlongWithForkSchedule(
+        ulong cancunTimestamp,
```

Comment:
```suggestion
        ulong eip4844Timestamp,
```
Generalise names for other chains where these EIPs might be enabled in a different fork